client/my-sites/domains/domain-management/gsuite/gsuite-add-users/new-user-form.jsx

```diff
@@ -30,39 +30,36 @@ class NewUserForm extends React.Component {
 	renderNameFieldset( firstName, lastName ) {
 		const { handleFieldChange, translate } = this.props;
 
+		const isFirstNameError = has( firstName, 'error' ) && null !== firstName.error;
+		const isLastNameError = has( lastName, 'error' ) && null !== lastName.error;
+
 		return (
 			<Fragment>
 				<FormFieldset>
 					<FormTextInput
-						isError={ has( firstName, 'error' ) && null !== firstName.error }
+						isError={ isFirstNameError }
 						maxLength={ 60 }
 						name="firstName"
 						onChange={ handleFieldChange.bind( this, 'firstName' ) }
 						onFocus={ this.handleFieldFocus.bind( this, 'First Name' ) }
 						placeholder={ translate( 'First Name' ) }
 						value={ firstName.value }
 					/>
-					<FormInputValidation
-						isError={ has( firstName, 'error' ) && null !== firstName.error }
-						isHidden={ ! has( firstName, 'error' ) || null === firstName.error }
-						text={ firstName.error || '\u00A0' }
-					/>
+					{ isFirstNameError && (
+						<FormInputValidation isError text={ firstName.error || '\u00A0' } />
```

Comment:
It looks like we no longer need those `\u00A0` default values: here `firstName.error` will never be null because of the check we perform with `isFirstNameError`.